### DISCUSSION TITLE
SMS service to Hong Kong

### DISCUSSION THREAD

**Discussion by kg-currenxie** [community/community#44634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm baffled how Gihub/Microsoft can't service SMS messages to Hong Kong..
I've lived here for 7 years, and I have NEVER ever NOT received an SMS (probably receiving TOTP messages almost daily, that's a few thousand messages over the years).
Not sure how you, or your SMS provider have issues delivering here. It's not the same as sending to mainland China.
🤷‍♂️ 🤦

**Comment by kenho811** [comment]
Exactly.

I have enabled 2FA in my github account and want to add SMS/Text Message as well.

However, Hong Kong is not one of the options.

I am disappointed to say the least.

**Comment by pakkinlau** [comment]
yeah that is crazy. 
but I can just set up my 2FA with authenticity app (I use google authenticator), which is better than SMS 2FA.

**Comment by fmchan** [comment]
same request here. i'm using github app to verify anyway...

**Comment by kensoncheung** [comment]
Needed for 2FA but NO Hong Kong option.

**Comment by mmw1984** [comment]
Dear GitHub,

Hong Kongers want SMS verification for them. 
It is simple, right?

**Comment by changmw** [comment]
It's really strange that China, Hong Kong and Macao are NOT in "Supported countries for SMS authentication", but Taiwan!!!

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication

Are there strange hurdles to do that? Politics regarding the definition of "country"? :)

I sincerely hope that GitHub could send SMS to at least Hong Kong(+852)!!! 

Thank you for your attention! May the Force and farces be with you!


Related:

2FA does NOT support SMS to Hong Kong (+852) · community · Discussion #102479
https://github.com/orgs/community/discussions/102479
